src/components/App.tsx

```diff
@@ -2210,7 +2211,7 @@ export class App extends React.Component<any, AppState> {
               let angle = (5 * Math.PI) / 2 + Math.atan2(y - cy, x - cx);
               if (event.shiftKey) {
                 angle += Math.PI / 16;
```

Comment:
Thanks for working on this!

```suggestion
                angle += SHIFT_LOCKING_ANGLE / 2;
```
It should be like this.

Reply:
@dai-shi no worries, Thank you for your feedback!
changed!